sql/sysviews_yb5/table_skew_p.sql

```diff
@@ -99,6 +99,7 @@ BEGIN
 
 -- Step 1: materialize table storage info using database object filters.
 
+  DROP TABLE IF EXISTS tsinfo;
```

Comment:
If a user has a regular table named `tsinfo` it's gonna be dropped! 😁
It's safer to use `pg_temp.tsinfo` in this case. But yes, good catch on missing DROP statement 👍